src/checkers/inference/InferenceOptions.java

```diff
@@ -122,6 +122,9 @@
     @Option("For inference, add debug on the port indicated")
     public static String debug;
 
+    @Option("If ture, don't insert solutions that are equivalent to the default ones back to the code.")
+    public static boolean ignoreDefaultAnnotations = true;
```

Comment:
Shouldn't this be false by default? Otherwise, there is no difference whether you pass the option or not?